### ISSUE TITLE
Error: Download manager failed to download from (a user_uploads URL)

### ISSUE THREAD

**Issue by borisyankov** [zulip/zulip-mobile#2115]
https://sentry.io/zulip/zulip-mobile/issues/501654438/

```
Error: Download manager failed to download from  https://machlearn.zulipchat.com/user_uploads/1129/uv_4kv70vLzFF7hLpBgEfAcN/mat8.PNG. Statu Code = 16
    at react-native-fetch-blob/index.prototype(index.js:329)
    count = args[0].count || count
    at react-native/Libraries/BatchedBridge/MessageQueue.__guard(MessageQueue.js:102)
    MessageQueue.prototype.__spy = info => {
    at react-native/Libraries/BatchedBridge/MessageQueue._inCall(MessageQueue.js:146)
    this.__guard(() => {
    at react-native/Libraries/BatchedBridge/MessageQueue.flushedQueue(MessageQueue.js:320)
    this._inCall--;
    at react-native/Libraries/BatchedBridge/MessageQueue.value(MessageQueue.js:142)
    return this.flushedQueue();
```

**Comment by gnprice** [comment]
This is likely related to #2618 .

**Comment by gnprice** [comment]
It looks like #3124 is a more detailed description of this issue. Marking as a duplicate of that one because that's what I think I would have done before the issue was fixed -- but thanks to @agrawal-d 's #4089, it is in fact now fixed 🙂
